### PR TITLE
Add Desktop version 

### PR DESCRIPTION
**Pull request by Kirabo19**
Add HTML, CSS for the desktop version 
Add media media queries for the functionality of all screen resize

### CODE REVIEW

File: css/port.css [hunk 1]
```diff
@@ -104,44 +111,135 @@ a {
...
+
+.section_text {
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+}
+
+/* media query for the header  */
+@media (max-width: 470px) {
```
Comment (sja-thedude) [inline css/port.css]: - [ ] Kindly remove this media query for the mobile version.  Make sure to use media queries for only Desktop breakpoint in the CSS file.

File: css/port.css [hunk 2]
```diff
@@ -104,44 +111,135 @@ a {
...
+  color: #091e42;
+}
+
+.section_text {
+  font-weight: 400px;
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+}
+
```
Comment (sja-thedude) [inline css/port.css]: - [ ] Your design is overlapping for the desktop version and the mobile version as you can see below. Kindly make sure to use media queries for the Desktop breakpoint in the CSS file. The value for the `min-width` property should be in the range of `768px` or wider.
Example: `@media screen and (min-width: 1080px) { // please note that it doesn’t have to be exactly 1080px, any value between 768 and 1440 is ok.`

https://user-images.githubusercontent.com/70269507/166839178-8caef090-7984-4247-a4cd-84df1eae3c71.mov

File: css/port.css [hunk 3]
```diff
@@ -236,142 +336,325 @@ section .aim ul li {
...
+  margin-top: 130px;
+}
+
+.pot3 {
+  top: 780px;
+  margin-top: 130px;
+}
+
+/* ---Portfolio----- */
+@media (max-width: 470px) {
```
Comment (sja-thedude) [inline css/port.css]: - [ ] Kindly remove this media query for the mobile version. Make sure to use media queries for only Desktop breakpoint in the CSS file.

File: css/port.css [hunk 4]
```diff
@@ -236,142 +336,325 @@ section .aim ul li {
...
-.about {
-  width: 375px;
-  margin: 70px auto;
-  background-color: #fff;
-  border-top-right-radius: 100px;
-  padding: 100px 0;
-  height: 1263px;
-}
+/* -----aboutus------------ */
+@media (max-width: 470px) {
```
Comment (sja-thedude) [inline css/port.css]: - [ ] Kindly remove this media query for the mobile version. Make sure to use media queries for only Desktop breakpoint in the CSS file.

File: css/port.css [hunk 5]
```diff
@@ -236,142 +336,325 @@ section .aim ul li {
...
 }
 
-.line {
-  background: #0d151e;
-  border-radius: 10px;
-  height: 5px;
-  width: 134px;
-  margin-top: 10px;
-  margin-left: 100px;
+@media (max-width: 470px) {
```
Comment (sja-thedude) [inline css/port.css]: - [ ] Kindly remove this media query for the mobile version. Make sure to use media queries for only Desktop breakpoint in the CSS file.

File: css/port.css [hunk 6]
```diff
@@ -104,44 +111,135 @@ a {
...
+
+/* media query for the header  */
+@media (max-width: 470px) {
+  .nav {
+    width: 50%;
+    font-size: 15px;
+    display: none;
+  }
+
+  .menu-icon {
```
Comment (sja-thedude) [inline css/port.css]: - [ ] Kindly make sure you use both Flexbox and Grid in order to layout the elements on the page. You have used flexbox so kindly use grid for both mobile and desktop version.

File: index.html [hunk 7]
```diff
@@ -1,276 +1,240 @@
...
+            I'd love to help with it
+          </p>
+          <div>
+            <form action="https://formspree.io/f/mvodnloy" method="post">
+              <input type="text" aria-required="true" maxlength="30" name="name" placeholder="Patrick Ndimukulaga">
+              <input type="email" aria-required="true" maxlength="30" name="email" placeholder="kapojames@gmail.com">
+              <textarea aria-required="true" maxlength="500" name="message" placeholder="Write your message here"></textarea>
+              <button class="btn" type="submit" >Get in Touch</button>
+            </form>
+          </div>
```
Comment (sja-thedude) [inline index.html]: MOBILE VERSION:

- [ ] Kindly follow the same placeholder name as given in the Figma design.
- [ ] Kindly make sure to follow the given Figma design and layout the elements according to it.
- [ ] Kindly make sure to reduce the extra space on the right side of the contact form as per the Figma design.
- [ ] Make sure to correctly place the background image as per the Figma design.
- [ ] Make sure the button is to the left side and not the centre as per the Figma design.
- [ ] Kindly follow the design as much as possible (e.g., font, colours, images, tests, margins) as per the Figma design.
- [ ] Make sure to follow the correct colour for placeholder text as per the Figma design.

- Figma design:
Screenshot 2022-05-05 at 1 41 40 AM

- Your design:
Screenshot 2022-05-05 at 2 10 44 AM

File: index.html [hunk 8]
```diff
@@ -1,276 +1,240 @@
...
+            if you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I'd love to help with it
+          </p>
+          <div>
+            <form action="https://formspree.io/f/mvodnloy" method="post">
+              <input type="text" aria-required="true" maxlength="30" name="name" placeholder="Patrick Ndimukulaga">
+              <input type="email" aria-required="true" maxlength="30" name="email" placeholder="kapojames@gmail.com">
+              <textarea aria-required="true" maxlength="500" name="message" placeholder="Write your message here"></textarea>
+              <button class="btn" type="submit" >Get in Touch</button>
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly follow the button interactions for hover, pressed, and enable as per the Figma design. 

Screenshot 2022-05-05 at 2 27 23 AM

File: index.html [hunk 9]
```diff
@@ -1,276 +1,240 @@
...
+            <img src="./images/Portfolio_Logo.png" alt="My Logo" class="logo"/>
+            <nav class="nav">
+                <ul>
+                    <li><a href="#" class="menu_li" >Portfolio</a></li>
+                    <li><a href="#" class="menu_li">About</a></li>
+                    <li><a href="#" class="menu_li">Contact</a></li>
+                </ul> 
+            </nav>
+            <img src="./images/Enabled.png" alt="menu" class="menu-icon"/>
+        </header>
```
Comment (sja-thedude) [inline index.html]: DESKTOP VERSION:

- [ ] Kindly make sure to add space in the header as per the Figma design. The list (Portfolio, About, Contact) is to the right end. Make sure that there is space on the right side.

Figma design:
Screenshot 2022-05-05 at 1 38 15 AM

Your design:
Screenshot 2022-05-05 at 2 30 32 AM

File: index.html [hunk 10]
```diff
@@ -1,276 +1,240 @@
...
+                <div class="section_header">I’m Patrick Ndimukulaga<br>Glad to see you!</div>
+                <p class="section_text">
+                  I’m a software developer! I can help you build a product , feature or
+                  website Look through some of my work and experience! If you like what
+                  you see and have a project you need coded, don’t hestiate to contact
+                  me.
+                </p>
+                <div class="media">
+                    <a href="#" class="media_p">LET'S CONNECT</a>
+                    <ul class="media_ul">
```
Comment (sja-thedude) [inline index.html]: MOBILE VERSION:

- [ ] Kindly reduce the space for the entire mobile version from the right side.
Figma design:
Screenshot 2022-05-05 at 1 40 19 AM

Your design:
Screenshot 2022-05-05 at 2 35 01 AM

File: index.html [hunk 11]
```diff
@@ -1,276 +1,240 @@
...
+              <div>
+                <h2 class="media_header">LET'S CONNECT</h2>
+                <ul class="media_ul">
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Github-icon.svg" alt="Github icon"/></a>
+                  </li>
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Linkedin-icon.svg" alt="Linkedin icon"/></a>
+                  </li>
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Vector.svg" alt="Vector icon"/></a>
+                  </li>
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Twitter-icon.svg" alt="Twitter icon"/></a>
```
Comment (sja-thedude) [inline index.html]: MOBILE VERSION:

- [ ] Kindly make sure the languages list is displayed as per the Figma design for the mobile version.
- [ ] Kindly make sure to reduce the spacing on the right side of the languages as per the Figma design as shown in the video.

Your design:


https://user-images.githubusercontent.com/70269507/166842668-dd4971d3-33c1-49ab-b003-bfe41f43da1f.mov



Figma design:
Screenshot 2022-05-05 at 1 41 16 AM

File: index.html [hunk 12]
```diff
@@ -1,276 +1,240 @@
...
+                      <ul class="ul">
+                        <li class="first-child">Uber</li>
+                        <li>Lead Developer</li>
+                        <li>2018</li>
+                      </ul>
+                    </div>
+                  </div>
+                  <p class="q-text">
+                    A daily selection of privately personalized reads; no accounts
+                    or sign-ups required.
```
Comment (sja-thedude) [inline index.html]: DESKTOP VERSION:

- [ ] Kindly follow the spacing to the right and left of the cards as per the Figma design. As you can see there is more spacing. Make sure to reduce as per the design.
- [ ] Kindly also make sure to follow the spacing in the cards. There should be space in between the image and text.

Figma design:
Screenshot 2022-05-05 at 1 39 10 AM

Your design:
Screenshot 2022-05-05 at 2 47 43 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Kirabo19, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job in implementing the desktop version.
✔️ All linters are passing
✔️ Correct git-flow.

## Required 
- [ ] Kindly make sure to update your readme file according to the project and add the necessary details.
- [ ] Kindly make sure to stick to the design as much as possible (e.g., font, colours, images, tests, margins) as per the Figma.
- [ ] _Check the comments under the review._

Note:- Most of the design is overlapping and it is because of the media query. Use media query for desktop version only and most of your design will fix.

## Optional suggestions
- [ ] Kindly make sure to fix your contact form as it is submitted without any values.
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**